### ISSUE TITLE
Hit test / scrolling does not work in scrolling example

### ISSUE THREAD

**Issue by fschutt** [servo/webrender#3898]
I recently noticed that the `scrolling` example doesn't work anymore, no matter what I do. For some reason, the rectangles don't scroll anymore. The `animation` example still works (so transform / opacity keys still seem to work).

I only get this output on running the scrolling example:

```
OpenGL version 3.3 (Core Profile) Mesa 19.2.8
Shader resource path: None
Device pixel ratio: 1.25
Loading shaders...
Entering event loop
Hit test results:

Hit test results:

Hit test results:

Hit test results:

Hit test results:

Hit test results:

Hit test results:

Hit test results:

Hit test results:

Hit test results:

```

Theoretically there should be a list of rectangles that have been hit (i.e. the rectangles that need to be scrolled), but... nothing.

## How to reproduce

```
git clone https://github.com/servo/webrender
cd webrender/examples
cargo run --bin scrolling
```

Then move the mouse cursor above the rectangles and see if scrolling works.

## Additional info

- webrender version: https://github.com/servo/webrender/commit/02be1053a833a5490800260540adeb19611fcb0d
- rustc version:

```
rustc 1.41.0 (5e1a79984 2020-01-27)
binary: rustc
commit-hash: 5e1a799842ba6ed4a57e91f7ab9435947482f7d8
commit-date: 2020-01-27
host: x86_64-unknown-linux-gnu
release: 1.41.0
LLVM version: 9.0
```

**Comment by kvark** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1625693

**Comment by atouchet** [comment]
This was fixed in #3963.
